VirtoCommerce.Storefront/Controllers/Api/ApiCartController.cs

```diff
@@ -137,37 +143,42 @@ public async Task<ActionResult> ChangeCartItemPrice([FromBody] ChangeCartItemPri
         // PUT: storefrontapi/cart/items?lineItemId=...&quantity=...
         [HttpPut("items")]
         [ValidateAntiForgeryToken]
-        public async Task<ActionResult> ChangeCartItem([FromBody] ChangeCartItemQty changeQty)
+        public async Task<ActionResult> ChangeCartItem([FromBody] ChangeCartItemQty[] changeQuantites)
         {
             EnsureCartExists();
 
             //Need lock to prevent concurrent access to same cart
             using (await AsyncLock.GetLockByKey(WorkContext.CurrentCart.Value.GetCacheKey()).LockAsync())
             {
                 var cartBuilder = await LoadOrCreateCartAsync();
-
-                var lineItem = cartBuilder.Cart.Items.FirstOrDefault(i => i.Id == changeQty.LineItemId);
-                if (lineItem != null)
+                foreach (var changeQty in changeQuantites)
```

Comment:
changeQuantity

Multi-line blocks of code should be separated with empty line from other code.